orttraining/orttraining/training_api/include/module.h

```diff
@@ -61,42 +64,42 @@ struct Module {
  public:
   // Initialize a module from an ORT inference session with loaded
   // training ONNX model and load parameters
-  Module(const std::string& /*train_model_path_or_bytes*/,
-         const std::unordered_map<std::string, std::shared_ptr<Parameter>>& /*parameters*/,
-         const std::optional<std::string>& /*eval_model_path_or_bytes*/) {
-    ORT_NOT_IMPLEMENTED("Not implemented.");
-  }
+  Module(const std::string& train_model_path_or_bytes,
+         std::unordered_map<std::string, std::shared_ptr<Parameter>>& parameters,
+         const std::optional<std::string>& eval_model_path_or_bytes = std::nullopt);
 
   // Return the trainable/nontrainable parameters
-  std::vector<std::shared_ptr<Parameter>> parameters() const {
-    return parameters_;
-  }
+  std::vector<std::shared_ptr<Parameter>> parameters() const;
```

Comment:
nit: Parameters()

Reply:
do we need this given we have NamedParameters?

Reply:
It exists to match pytorch's parameters() v/s named_parameters(), but we can reassess and remove if required